frontend/src/reducers/cuisines.reducer.js

```diff
@@ -0,0 +1,28 @@
+import { REQUEST_CUISINES, RETRIEVED_CUISINES } from '../actions';
+
+const cuisinesReducer = (
```

Comment:
`redux-actions` offers a really nice helper called `handleActions` for building reducers.